robotidy/transformers/OrderTags.py

```diff
@@ -0,0 +1,83 @@
+from robot.api.parsing import ModelTransformer, Tags, Token
+
+
+class OrderTags(ModelTransformer):
+    """
+    Order tags inside Keywords and Test Cases.
+
+    Tags are ordered like this:
+
+        *** Test Cases ***
+        Tags Upper Lower
+            [Tags]    ba    Ab    Bb    Ca    Cb    aa
+            My Keyword
+
+        *** Keywords ***
+        My Keyword
+            [Tags]    ba    Ab    Bb    Ca    Cb    aa
+            No Operation
+
+    To:
+
+        *** Test Cases ***
+        Tags Upper Lower
+            [Tags]    aa    Ab    ba    Bb    Ca    Cb
+            My Keyword
+
+        *** Keywords ***
+        My Keyword
+            [Tags]    aa    Ab    ba    Bb    Ca    Cb
+            No Operation
+
+    Default order can be changed using following parameters:
+      - ``case_sensitive = False``
+      - ``reverse = False``
+
+    See https://robotidy.readthedocs.io/en/latest/transformers/OrderTags.html for more examples.
+    """
+    ENABLED = False
+
+    def __init__(self, case_sensitive=False, reverse=False):
+        self.key = self.get_key(case_sensitive)
+        self.reverse = reverse
+        self.separator = '    '
```

Comment:
Use global separator (that can be set through cli):
```
self.formatting_config.separator
```
but since you only use self.separator for creating tokens you can save save some time by creating it just once and then adding it whenver necessary:
```
self.separator =  Token(Token.SEPARATOR, self.formatting_config.separator)
```
and then
```
            result += (
                self.separator,
                Token(Token.TAGS, tag),
            )
```

Reply:
self.formatting_config.separator is not accessible in transformer init, I used in function

Reply:
Did you try to use it? I know that IDE suggest it's not available but it's injected in runtime - I know it's not the best practise ( I will fix it) but for now you should be able to use self.formatting_config.separator without any issues.

Reply:
I tried, I got:
`AssertionError: robotidy exit code: 1 does not match expected: 0. Exception description: Importing 'robotidy.transformers.OrderTags' failed: Creating instance failed: AttributeError: 'OrderTags' object has no attribute 'formatting_config'`

Reply:
Ok. I see that you managed to use formatting_config in other method so I guess it's fine.